### PR TITLE
Agenda follow up meeting 20180619

### PR DESCRIPTION
**Pull request by teresanunez**

### CODE REVIEW

File: 20180619/AGENDA.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Sardana Follow-up Meeting - 2018/06/19
+
+To be held on 2018/06/19 at 14:00
+
+# Agenda
+1. Urgent issues/problems
+2. Preparation of the Sardana Documentation Camp (Alba-July18)
```
Comment (reszelaz) [inline 20180619/AGENDA.md:7]: For the point 2. What do you think if we quickly present the following things:
* Present structure of the current Sardana documentation (hirerarchy of chapters, important links between them, which chapters are empty, etc.).
* Present structure of the Sardana-DESY documentation (the same as previous point witht focus on the parts of the docs which are generic and could be moved to the Sardana documentation).
* Best practices when writing documentation (I found [Writing Great Documentation](http://www.writethedocs.org/guide/#writing-great-documentation) from www.writethedocs.org but couldn't read it yet), for sure there are more sources in the internet.
* Present a documentation strucuture of some other well documented project (maybe we could take a look on Tango which documentation was recently rewritten).

Or if you see other interesting things, just propose.

Comment (teresanunez) [reply 20180619/AGENDA.md:7]: I think the points you propose are fine.
With respect to the DESY documentation, I have had a look, and it is actually so much dependent
on our system, that I do not see anything that we could use as general ... not a whole section ...
 perhaps some paragraphs, but nothing really big.

**Comment by reszelaz** [comment]
@teresanunez and @amilan volunteered to present Sardana-DESY documentation and "How to write good documentation" (WriteTheDocs, etc.) respectivelly.
@daneos if you want you can grab one of the resting points, but no pressure - I can manage with both.

**Comment by daneos** [comment]
I can take a look on Tango documentation, and also find some other projects with good docs.

**Comment by reszelaz** [comment]
Great Grzegorz! I then will review the current Sardana docs! Thanks to all!

**Comment by teresanunez** [comment]
I merge this, so we have it available in the official repo during the meeting.

**Pull request merged**